### DISCUSSION TITLE
What's going on with create next-app ??

### DISCUSSION THREAD

**Discussion by rgioeli** [vercel/next.js#51632] [Help] [answered]
### Summary

_**Multiple**_ warning appear when creating a fresh app with **create next-app**. I only posted this one section of the logs because of sensitive information. I tried deleting node_modules and switching the version of next to 13.3 but still run into a plethora of warnings.

![image](https://github.com/vercel/next.js/assets/45113824/74edeede-2bca-4fdf-a90c-5a11330d00a4)

**Comment by icyJoseph** [comment, marked as answer]
Hi,

This is a warning coming straight up from WebPack.

The problem is that different Operating System, give file casing different meaning. 

> Windows file system treats file and directory names as case-insensitive. FOO.txt and foo.txt will be treated as equivalent files.
> Linux file system treats file and directory names as case-sensitive. FOO.txt and foo.txt will be treated as distinct files.

So, really having such difference in files, is just asking for trouble, might be a good idea to listen to the warning. 

A build might succeed in a Windows machine, but when deployed to a Linux environment, it won't.

**Reply by rgioeli** [reply]
Gotcha. I went to go try and use **_yarn create next-app_** on a different computer and didn't receive these warnings. Any idea why one computer might create the frame for a new next app without any warnings and the other computer log out various warnings?

**Reply by icyJoseph** [reply]
Same OS on both? Which one? I imagine some temporary files were created that caused this warning? Strange...
